EShop.Web/Views/Home/Index.cshtml

```diff
@@ -4,5 +4,5 @@
 
 <div class="text-center">
     <h1 class="display-4">Welcome</h1>
-    <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
+    <p>Prima Mea Schimbare</p>
```

Comment:
Te rog sa schimbi literele mari din mea si schimbare in litere mici!

Reply:
gata